VoxEU.js

```diff
@@ -115,5 +121,5 @@ var testCases = [
 		"url": "http://www.voxeu.org/search/node/Are%20migrants%20paid%20more%3F",
 		"items": "multiple"
 	}
-]
-/** END TEST CASES **/
\ No newline at end of file
+];
```

Comment:
no

Reply:
I thought I had caught all of those. Sorry. The updated linter rule from https://github.com/zotero/translators/pull/1958 will prevent more of these happening.